yml/dp03_10yr.yaml

```diff
@@ -13,6 +13,7 @@ tables:
   primaryKey: "#SSObject.ssObjectId"
   mysql:engine: MyISAM
   mysql:charset: utf8mb4
+  tap:table_index: 0
```

Comment:
I prefer not to use `table_index=0` on purpose.  That way places where it's accidentally defaulted to 0 stand out in the end product.